test/lib/global_uid_test.rb

```diff
@@ -198,6 +198,21 @@ def table_exists?(connection, table)
         refute Person.const_get(:HABTM_Account).global_uid_disabled
       end
     end
+
+    describe "Setting a start ID" do
+      it "creates records after the given start ID, respecting server offsets" do
+        CreateWithGlobalUIDAndCustomStart.up
+
+        GlobalUid::Base.with_servers do |server|
+          assert table_exists?(server.connection, 'with_global_uid_and_custom_start_ids'), 'Table should exist'
+        end
+
+        # An InvalidIncrementException would be raised if the server offset wasn't respected
+        25.times { assert_operator WithGlobalUIDAndCustomStart.create!.id, :>, 1000 }
```

Comment:
I don’t understand the assertion. The table is created with a start_id of 10_000, right?

Reply:
You're completely right. I was missing a 0 🤦